packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -639,31 +678,63 @@ class _TextSelectionHandleOverlayState
       point.dy.clamp(0.0, viewport.height),
     );
 
+    final Offset handleAnchor = widget.selectionControls.getHandleAnchor(
+      type,
+      widget.renderObject.preferredLineHeight,
+    );
+    final Size handleSize = widget.selectionControls.getHandleSize(
+      widget.renderObject.preferredLineHeight,
+    );
+    final Rect handleRect = Rect.fromLTWH(
+      // Put handleAnchor on top of point
+      point.dx - handleAnchor.dx,
+      point.dy - handleAnchor.dy,
+      handleSize.width,
+      handleSize.height,
+    );
+
+    // Make sure the GestureDetector is big enough to be easily interactive.
+    final Rect interactiveRect = getInteractiveRect(handleRect);
+    final double paddingHorizontal = handleRect.width < interactiveRect.width
+      ? (interactiveRect.width - handleRect.width) / 2
+      : 0;
+    final double paddingVertical = handleRect.height < interactiveRect.height
+      ? (interactiveRect.height - handleRect.height) / 2
+      : 0;
+
     return CompositedTransformFollower(
       link: widget.layerLink,
+      offset: interactiveRect.topLeft,
       showWhenUnlinked: false,
       child: FadeTransition(
         opacity: _opacity,
-        child: GestureDetector(
-          dragStartBehavior: widget.dragStartBehavior,
-          onPanStart: _handleDragStart,
-          onPanUpdate: _handleDragUpdate,
-          onTap: _handleTap,
-          child: Stack(
-            // Always let the selection handles draw outside of the conceptual
-            // box where (0,0) is the top left corner of the RenderEditable.
-            overflow: Overflow.visible,
-            children: <Widget>[
-              Positioned(
-                left: point.dx,
-                top: point.dy,
+        child: Container(
+          alignment: Alignment.topLeft,
+          width: interactiveRect.width,
+          height: interactiveRect.height,
+          child: GestureDetector(
+            behavior: HitTestBehavior.translucent,
+            dragStartBehavior: widget.dragStartBehavior,
+            onPanStart: _handleDragStart,
+            onPanUpdate: _handleDragUpdate,
+            onTap: _handleTap,
+            // The above GestureDetector handles all interaction, so interaction
+            // further down the tree can be ignored.
+            child: IgnorePointer(
```

Comment:
Is this really needed? If we're defending against the possibility that buildHandle will include a gesture detector, then, if the handle actually _does_ include a gesture detector, the developer is going to be confused.

Reply:
Yeah you're right, I think I added this during one of my earlier iterations on this and never realized it wasn't needed anymore.